public/app/plugins/datasource/loki/addToQuery.ts

```diff
@@ -57,6 +57,23 @@ export function addParserToQuery(query: string, parser: string): string {
   }
 }
 
+/**
+ * Adds filtering for pipeline errors to existing query. Useful for query modification for hints.
+ * It uses LogQL parser to find parsers and adds pipeline errors filtering after them.
+ *
+ * @param query
+ */
+
```

Comment:
I think this line should be removed in order to best map the comment to the function.